### PR TITLE
[All hosts] Fill in empty/blank cells in tables

### PR DESCRIPTION
**Pull request by alison-mk**
Improving screenreader accessibility. [Internal work item](https://office.visualstudio.com/OC/_workitems/edit/6192823).

### CODE REVIEW

**Comment by alison-mk** [comment]
@ElizabethSamuel-MSFT No need to review today! Can wait until next week.

File: docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Table 2 lists the limits and describes the differences in the portion of the ite
 
 **Table 2. Limits on the size of the item body evaluated**
 
-||Outlook rich client|Outlook on mobile devices|Outlook on the web|
+|&mdash;|Outlook rich client|Outlook on mobile devices|Outlook on the web|
```
Comment (alison-mk) [inline docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md:48]: @ElizabethSamuel-MSFT I found a few tables in this style, where the first (top left) cell was empty. Curious what you think of the approach I chose.

File: docs/concepts/resource-limits-and-performance-optimization.md [hunk 2]
```diff
@@ -76,19 +76,19 @@ For each event that the Telemetry Log tracks for an add-in, there is a date/time
 
 |**Date/Time**|**Event ID**|**Severity**|**Title**|**File**|**ID**|**Application**|
```
Comment (ElizabethSamuel-MSFT) [inline docs/concepts/resource-limits-and-performance-optimization.md:77]:
```suggestion
|Date/Time|Event ID|Severity|Title|File|ID|Application|
```

File: docs/concepts/resource-limits-and-performance-optimization.md [hunk 3]
```diff
@@ -76,19 +76,19 @@ For each event that the Telemetry Log tracks for an add-in, there is a date/time
...
 |:-----|:-----|:-----|:-----|:-----|:-----|:-----|
-|10/8/2012 5:57:10 PM|7||add-in manifest downloaded successfully|Who's Who|69cc567c-6737-4c49-88dd-123334943a22|Outlook|
-|10/8/2012 5:57:01 PM|7||add-in manifest downloaded successfully|LinkedIn|333bf46d-7dad-4f2b-8cf4-c19ddc78b723|Outlook|
+|10/8/2012 5:57:10 PM|7|*Not applicable*|add-in manifest downloaded successfully|Who's Who|69cc567c-6737-4c49-88dd-123334943a22|Outlook|
+|10/8/2012 5:57:01 PM|7|*Not applicable*|add-in manifest downloaded successfully|LinkedIn|333bf46d-7dad-4f2b-8cf4-c19ddc78b723|Outlook|
 
 The following table lists the events that the Telemetry Log tracks for Office Add-ins in general.
 
 |**Event ID**|**Title**|**Severity**|**Description**|
```
Comment (ElizabethSamuel-MSFT) [inline docs/concepts/resource-limits-and-performance-optimization.md:84]:
```suggestion
|Event ID|Title|Severity|Description|
```

File: docs/develop/json-manifest-overview.md [hunk 4]
```diff
@@ -74,19 +74,19 @@ The base manifest properties specify characteristics of the add-in that *any* ty
 
 |JSON property|Purpose|XML element(s)|Comments|
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/json-manifest-overview.md:75]: Per the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=25525), we should use the plural form of the noun when the number is variable. I've also found that the narrator read this as "element ess".
```suggestion
|JSON property|Purpose|XML elements|Comments|
```

File: docs/develop/json-manifest-overview.md [hunk 5]
```diff
@@ -99,20 +99,20 @@ The following table shows a mapping of some high level child properties of the "
...
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/json-manifest-overview.md:100]:
```suggestion
|JSON property|Purpose|XML elements|Comments|
```

File: docs/develop/json-manifest-overview.md [hunk 6]
```diff
@@ -99,20 +99,20 @@ The following table shows a mapping of some high level child properties of the "
...
-| "runtimes"  | Configures various kinds of add-ins that have little or no UI, such as custom function-only add-ins and [function commands](../design/add-in-commands.md#types-of-add-in-commands). | **\<Runtimes\>**. **\<FunctionFile\>**, and **\<ExtensionPoint\>** (of type CustomFunctions) |  |
-| "autoRunEvents" | Configures an event handler for a specified event. | **\<Event\>** and **\<ExtensionPoint\>** (of type Events) |  |
+| "runtimes"  | Configures various kinds of add-ins that have little or no UI, such as custom function-only add-ins and [function commands](../design/add-in-commands.md#types-of-add-in-commands). | **\<Runtimes\>**. **\<FunctionFile\>**, and **\<ExtensionPoint\>** (of type CustomFunctions) |*None.* |
+| "autoRunEvents" | Configures an event handler for a specified event. | **\<Event\>** and **\<ExtensionPoint\>** (of type Events) |*None.* |
 
 ##### "ribbons" table
 
 The following table maps the child properties of the anonymous child objects in the "ribbons" array onto XML elements in the current manifest. 
 
 |JSON property|Purpose|XML element(s)|Comments|
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/json-manifest-overview.md:113]:
```suggestion
|JSON property|Purpose|XML elements|Comments|
```

File: docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md:48]:
```suggestion
||Outlook rich client|Outlook on mobile devices|Outlook on the web|
```

Comment (ElizabethSamuel-MSFT) [reply docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md:48]: Per the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=36413), we shouldn't use an em dash either. However, tables where the first column has no title but represents disparate info are considered data tables and this is expected (see [contributor guide](https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?toc=%2Fhelp%2Fplatform%2Ftoc.json&bc=%2Fhelp%2Fplatform%2Fbreadcrumb%2Ftoc.json&branch=main#data-matrix-tables)). As such, it should be ok to leave it blank. That said, I don't know what the narrator experience is.

File: docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md [hunk 7]
```diff
@@ -55,7 +55,7 @@ Table 3 lists the limits and describes the differences in the matches that each
 
 **Table 3. Limits on the matches returned**
 
-||Outlook rich client|Outlook on the web or mobile devices|
+|&mdash;|Outlook rich client|Outlook on the web or mobile devices|
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/limits-for-activation-and-javascript-api-for-outlook-add-ins.md:58]:
```suggestion
||Outlook rich client|Outlook on the web or mobile devices|
```

File: docs/outlook/outlook-on-send-addins.md [hunk 8]
```diff
@@ -308,7 +308,7 @@ By default, the on-send policy is disabled. Administrators can enable the on-sen
 
 Add-ins for Outlook on Mac that use the on-send feature should run for any users who have them installed. However, if users are required to run the add-in to meet compliance standards, then the following mailbox setting must be applied on each user's machine. This setting or key is CFPreference-compatible, which means that it can be set by using enterprise management software for Mac, such as Jamf Pro.
 
-||Value|
+|&mdash;|Value|
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/outlook-on-send-addins.md:311]:
```suggestion
||Value|
```

**Comment by alison-mk** [comment]
Thanks for reviewing @ElizabethSamuel-MSFT! I applied your suggestions and then removed the bold formatting around a couple more table head rows within these files.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**